collect_app/src/main/java/org/odk/collect/android/preferences/PreferencesActivity.java

```diff
@@ -38,9 +40,14 @@ protected void onCreate(Bundle savedInstanceState) {
 
         setTitle(R.string.general_preferences);
         if (savedInstanceState == null) {
+            boolean adminMode = getIntent().getBooleanExtra(INTENT_KEY_ADMIN_MODE, false);
+            boolean openMapsPrefs = getIntent().getBooleanExtra("openMapsPrefs", false);
```

Comment:
Was this extra just for the version where the whole maps preference screen was opened from widgets, by any chance?

Reply:
Yes, that's right!  I've removed it.